### ISSUE TITLE
Customize/ Modify icon

### ISSUE THREAD

**Issue by foxmountain-vulp** [Templarian/MaterialDesign#7226]
### Before submitting a new icon contribution, please confirm the following:

- [X] I have [searched all issues](https://github.com/Templarian/MaterialDesign/issues?q=is%3Aissue) to make sure there isn't a request for this icon.
- [X] I have [searched the current library](https://pictogrammers.com/library/mdi) to make sure the icon doesn't already exist.
- [X] I understand that [brand icons are no longer being added](https://pictogrammers.com/docs/library/mdi/guides/brand-icons/) and I can use [SimpleIcons](https://simpleicons.org/) for my brand icon needs.
- [X] I have followed the [Material guidelines](https://material.io/design/iconography/system-icons.html) for system icons.

### Tell us about your contribution.

There is no icon for 'customize' or 'modify.

### Share a preview of your contribution.

![image](https://github.com/Templarian/MaterialDesign/assets/119973635/7c331648-94fc-4f2f-94b8-5ee7bd0e04d4)

### Upload the icon source.

[pencil-wrench.zip](https://github.com/Templarian/MaterialDesign/files/12516123/pencil-wrench.zip)

**Comment by MrGrigri** [comment]
I get that these are the actual sizes of the icons individually, but it seams off because the wrench is smaller than the pincil.

**Comment by foxmountain-vulp** [comment]
I agree with your feedback. I changed the size to the same thickness as the wrench. If made even smaller the icon looks like an 'edit-wrench' icon.

![image](https://github.com/Templarian/MaterialDesign/assets/119973635/76146c59-0eeb-46fa-9e13-43c053fa6dd4)

![pencil-wrench](https://github.com/Templarian/MaterialDesign/assets/119973635/67b4d1cc-4b5a-4115-a53d-1397a0b7cda1)
